### ISSUE TITLE
Feature Request: Command to Recompile Currently Installed LTS Kernel

### ISSUE THREAD

**Issue by timkofu** [xanmod/linux#319]
Hello,

Thank you for this great project.

I have Xanmod LTS installed on Ubuntu 22.04 LTS and it's working well, but I can get a significant performance boost if I recompile it with certain (C/CXX)FLAGS set.

Is there an already existing command that recompiles `Xanmod LTS` as-is (no configuration change whatsoever) with `CFLAGS="-march=native -Ofast"` set?

**Comment by FerdiScholten** [comment]
You **must** change the kernel configuration to set these options. 

There is an entry in there to set the desired processor type.
This will give (a little) bit of extra performance in some kernel operations.

Setting the O  level will mostly have no effect at all (according to benchmarks from Phoronix who did just that). It will however change the size of your kernel (usually it gets bigger) Or even worse it can potentially break some kernel functionality. This is also why the O level when applicable can be set in the kernel configuration.

The kernel compilation process totally ignores your (C/CXX)FLAGS setting on your system. If you want to use environment variables to change compilation setting you need these: [https://www.kernel.org/doc/html/latest/kbuild/kbuild.html#environment-variables]https://www.kernel.org/doc/html/latest/kbuild/kbuild.html#environment-variables

Long story short, create your own modified config based on Xanmod and use that to recompile.

You might get a bit better performance but it will not be significant! It will most likely be more efficient and the perfomance gain will only be seen when benchmarking and probably not with real life applications and usage.

**Comment by timkofu** [comment]
@FerdiScholten thanks.
